packages/web-components/fast-components/src/slider/fixtures/base.html

```diff
@@ -1,342 +1,340 @@
+        <button
+            style="width: 90px; margin: 10px;"
+            onclick="const slider = document.getElementById('switcher'); slider.getAttribute('orientation') === 'horizontal' ? slider.setAttribute('orientation', 'vertical') : slider.setAttribute('orientation', 'horizontal');"
+        >
+            Toggle orientation
+        </button>
+        <button
+            style="width: 90px; margin: 10px;"
+            onclick="const slider = document.getElementById('switcher'); const currentVal = Number(slider.value); slider.value = (currentVal + 10);"
+        >
+            Change slider val
+        </button>
+        <fast-slider id="switcher" orientation="vertical" min="0" max="100" step="10">
+            <fast-slider-label position="0">
+                0&#8451;
             </fast-slider-label>
-            <fast-slider-label position="-2">
-                -2
+            <fast-slider-label position="10">
+                10&#8451;
             </fast-slider-label>
-            <fast-slider-label position="-1">
-                -1
+            <fast-slider-label position="90">
+                90&#8451;
             </fast-slider-label>
-            <fast-slider-label position="0">
-                0
+            <fast-slider-label position="100">
+                100&#8451;
+            </fast-slider-label>
+        </fast-slider>
+    </div>
+    <!-- No max or min -->
+    <h2>Defaults</h2>
+    <fast-slider></fast-slider>
+
+    <!-- Margin offsets -->
+    <h2>horizontal with left offset</h2>
+    <fast-slider style="margin-left: 100px;"></fast-slider>
+
+    <!-- Margin offsets vertical -->
+    <h2>Vertical with top offset</h2>
+    <fast-slider
+        orientation="vertical"
+        style="margin-left: 100px; height: 120px;"
+    ></fast-slider>
+
+    <!-- Negative positions -->
+    <h2>Negative positions</h2>
+    <fast-slider min="-3" max="3" step="1">
+        <fast-slider-label position="-3">
+            -3
+        </fast-slider-label>
+        <fast-slider-label position="-2">
+            -2
+        </fast-slider-label>
+        <fast-slider-label position="-1">
+            -1
+        </fast-slider-label>
+        <fast-slider-label position="0">
+            0
+        </fast-slider-label>
+        <fast-slider-label position="1">
+            1
+        </fast-slider-label>
+        <fast-slider-label position="2">
+            2
+        </fast-slider-label>
+        <fast-slider-label position="3">
+            3
+        </fast-slider-label>
+    </fast-slider>
+
+    <!-- Min above 0 -->
+    <h2>Min value greater than 0</h2>
+    <div style="width: 160px;">
+        <fast-slider min="5" max="15" step="1" value="10" style="min-width: unset;">
+            <fast-slider-label hide-mark="true" position="5">
+                5
             </fast-slider-label>
-            <fast-slider-label position="1">
-                1
+            <fast-slider-label hide-mark="true" position="15">
```

Comment:
Why are these values changing?

Reply:
🤔 that's weird. Must be a formatter issue.

Reply:
Yep, must be, double checked it against the original and it looks correct.